provider/identityprovider/htpasswd.go

```diff
@@ -119,3 +120,27 @@ func uniqueUsernameValidator() validator.List {
 		}
 	})
 }
+
+func HTPasswdUserListToStringMaps(ctx context.Context, users []HTPasswdUser,
```

Comment:
The fuction can be private as it is used only from within the package
```suggestion
func hTPasswdUserListToStringMaps(ctx context.Context, users []HTPasswdUser,
```